src/components/SourceInfoPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import { ReactComponent as SourceInfoPageSvg } from "../assets/source-info.svg";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  img: {
+    display: "flex",
+    flexDirection: "column",
+    margin: "40px",
+  },
+  description: {
+    marginTop: "38px",
+    maxWidth: "700px",
+    textAlign: "center",
+  },
+}));
+
+export const SourceInfoPage = () => {
+  const classes = useStyles();
+  return (
+    <>
+      <Typography variant="h5">
+        The product scoring is based on industry research
+      </Typography>
+      <Typography className={classes.description}>
+        The product scoring is based on industry research on how sustainability
+        is scored in cosmetic and fashion research. Cosmelo aimed to develop a
+        comprehensive score that judges sustainability of the entire life cycle
+        of products based on company policies and initiatives. The
+        sustainability categories are based on the 2018 release of the BPC
+        (Beauty and Personal Care) Sustainability Rating, simplified to 4
```

Comment:
Can I request the link to the 2018 BPC Sustainability Rating doc be embedded? 

Link: https://www.sustainabilityconsortium.org/wp-content/uploads/2018/04/TSC%C2%AE-BPC-Product-Sustainability-Rating-System.pdf

Reply:
Can you clarify what that means? do you mean you would like the text "BPC
        (Beauty and Personal Care) Sustainability Rating" to be a hyperlink?

Reply:
I was thinking we could have the link be somewhere on the page, either like how we had it in our symposium poster or as a link with a footnote or something. Just thinking about transparency, thought we should provide a link/source when we have one.